foodplaner/users/admin.py

```diff
@@ -1,3 +1,9 @@
+"""
+Automatic admin interface. 
+It reads metadata from the models to provide a quick, model-centric interface where trusted users can manage content. 
+The admin’s recommended use is limited to an organization’s internal management tool.
+"""
+
```

Comment:
Hier ist die generische Fassung wahrscheinlich angebracht. Allerdings bitte trotzdem auf die Leerzeile nach der kurzen Beschreibung (keine Überschrift!) in der ersten Zeile achten.